package.json

```diff
@@ -27,11 +28,14 @@
 		"@octokit/request-error": "6.1.5",
 		"@octokit/webhooks": "13.3.0",
 		"@opentelemetry/api": "1.9.0",
+		"@opentelemetry/api-logs": "^0.57.1",
```

Comment:
@sakit0 We have stopped using `^` in this issue https://github.com/giselles-ai/giselle/issues/293, so please make the necessary adjustments accordingly!

Reply:
fix!!
https://github.com/giselles-ai/giselle/pull/306/commits/81b86f946cd2fb0449df72ca172320ad15349f5f